### PR TITLE
Adds a totally innocent Giant Medical Dog Donor Skin

### PR DESCRIPTION
**Pull request by fluffe9911**
### Intent of your Pull Request
To make the server worse overall
#### Changelog

:cl:  
rscadd: A new donor skin thats a giant medical dog with no connections to lewdness what so ever!
/:cl:

dontblamemeblamemyupbringing

### CODE REVIEW

File: yogstation/code/modules/donor/borg_ai_skin_datums.dm [hunk 1]
```diff
@@ -190,6 +190,13 @@
 	module_locked = "Medical"
 	owner = null
 
+/datum/borg_skin/medical_dog //completely innocent and child friendly
+	name = "Giant Medical Dog"
+	icon = 'yogstation/icons/mob/widedonorrobots.dmi'
+	icon_state = "medihound"
+	module_locked = "Medical"
+	owner = null
```
Comment (Moja10) [inline yogstation/code/modules/donor/borg_ai_skin_datums.dm:198]: This is clean code

**Review by Moja10 — APPROVED** [review]
Great change, love the skin!

File: yogstation/code/modules/donor/borg_ai_skin_datums.dm (hunk 1 above)
Comment (fluffe9911) [reply yogstation/code/modules/donor/borg_ai_skin_datums.dm:198]: yes

**Comment by patpol4** [comment]
Upvoting because I want to see Moja get eaten by Xantam

**Comment by yacabo** [comment]
image

**Review by SomeguyManperson — APPROVED** [review]
you're all going to brazil

File: yogstation/code/modules/donor/borg_ai_skin_datums.dm [hunk 2]
```diff
@@ -190,6 +190,13 @@
 	module_locked = "Medical"
 	owner = null
 
+/datum/borg_skin/medical_dog //completely innocent and child friendly
+	name = "Giant Medical Dog"
```
Comment (SomeguyManperson) [inline yogstation/code/modules/donor/borg_ai_skin_datums.dm:194]:
```suggestion
	name = "Vorgi"
```

Comment (fluffe9911) [reply yogstation/code/modules/donor/borg_ai_skin_datums.dm:194]: this is a completely innocent dog sprite that has nothing to do with the v word totally yes sir re

**Comment by JamieD1** [comment]
Didnt even add it right, where is the way for anyone to change into it?

**Comment by JamieD1** [comment]
Actually no. Admin only? Fuck yeah

**Review by JamieD1 — APPROVED** [review]

**Comment by MommaDragon** [comment]
HOW DID IT GET MERGED

**Pull request merged**